server/modifiers.py

```diff
@@ -302,3 +296,62 @@ def add_or_remove_brackets(self) -> Self:
         self.modified_count += 1
 
         return self
+
+    def _get_replacements(self) -> ReplaceData:
+        """A modifier that modifies the modified contents.
+
+        This method is to convert all of the code modifications
+        into a format that can be sent as an EventResponse to
+        the client.
+
+        Returns:
+            The converted replacement data.
+        """
+        replacements = []
+
+        current_position = 0
+        value = ""
+        deletes = 0
+        for input_line, output_line in zip(self.file_contents, self.modified_contents):
+            for diff in difflib.ndiff(input_line, output_line):
+                if diff[0] == "-":
+                    # Removed values should always have the value set to ""
+                    replacements.append(
+                        {"from": current_position - deletes, "to": (current_position + 1) - deletes, "value": ""}
+                    )
+                    deletes += 1
+
+                if diff[0] == "+":
+                    value = diff[-1]
+                    replacements.append(
+                        {"from": current_position - deletes, "to": current_position - deletes, "value": value}
+                    )
```

Comment:
I don't even think we need to have `value = diff[-1]` anymore since value is only used in this context. You can remove line 325 as well as 313, then replace 327 `"value": value` with `"value": diff[-1]`